Gemfile

```diff
@@ -6,3 +6,4 @@ gem 'rspec', '~> 3.5.0'
 gem 'guard'
 gem 'guard-rspec', require: false
 gem 'debug'
+gem 'i18n'
```

Comment:
This is a dependency of the gem. It needs to go in the bible_bot.gemspec file, so that consumers of gem install the appropriate dependencies.

```ruby
spec.add_dependency 'i18n'
```

```suggestion
```

Reply:
For example:
https://github.com/rails/rails/blob/9c8390032c85df23d973432b67d025df05c838f7/actionmailer/actionmailer.gemspec#L36-L42

Reply:
Ah, right. https://github.com/LittleLea/bible_bot/pull/18/commits/a6b05dc9d2f36793f4fbde4cd46152d10eb10863